tutorials/CommandsArguments.md

```diff
@@ -0,0 +1,58 @@
+This is an in-depth tutorial for **Klasa's Usage arguments**, explaining all kinds of arguments and what you can do with them, as well as how they're used in the framework's backends.
+
```

Comment:
Big and complicated sentence right at the start of a beginner's tutorial. I'd split it into two sentences.